vars/Pipeline.groovy

```diff
@@ -149,6 +149,11 @@ def call() {
               echo '  LATEST_PRODUCT_BUILD_ARTIFACTS_API: ${LATEST_PRODUCT_BUILD_ARTIFACTS_API}' >> ${JOB_CONFIG_YAML_PATH}
               echo '  TEST_MODE: ${TEST_MODE}' >> ${JOB_CONFIG_YAML_PATH}
               echo '  runOnBranch: "false"' >> ${JOB_CONFIG_YAML_PATH}
+              echo '  PRODUCT_LATEST_URL: ${PRODUCT_LATEST_URL}' >> ${JOB_CONFIG_YAML_PATH}
+              echo '  PRODUCT_LATEST_BRANCH: ${PRODUCT_LATEST_BRANCH}' >> ${JOB_CONFIG_YAML_PATH}
```

Comment:
What is a PRODUCT_LATEST_BRANCH? What is a PRODUCT_LATEST_URL?

Reply:
This is for WUM. For the WUM its taking Product_apim support branch as product pack. Named git support URL as "PRODUCT_LATEST_URL" and branch as "PRODUCT_LATEST_BRANCH"